### ISSUE TITLE
Bug: $subtotal not defined

### ISSUE THREAD

**Issue by growdev** [strangerstudios/paid-memberships-pro#302]
In the latest release (1.8.8) on this line:
https://github.com/strangerstudios/paid-memberships-pro/blob/dev/classes/class.memberorder.php#L579

Looks like `$amount` was changed to `$subtotal`. This throws a warning:

```
PHP Notice:  Undefined variable: subtotal in /Users/daniel/sites/test/wp-content/plugins/paid-memberships-pro/classes/class.memberorder.php on line 579
```

Should probably be `$this->subtotal`

**Comment by eighty20results** [comment]
This should be resolved in the latest release (1.8.8.3). Let us know if that's not the case for you.

**Comment by growdev** [comment]
It is, thanks!  I was just testing out the ConvertKit PMP integration.
